rfcs/017 - Cluster Data Storage RFC.md

```diff
@@ -0,0 +1,221 @@
+# 017 - Cluster Data Storage RFC
+
+## Current Status
+
+### Proposed
+
+2021-04-01
+
+### Accepted
+
+YYYY-MM-DD
+
+#### Approvers
+
+- Luis De Pombo <luis@alantechnologies.com>
+- Alejandro Guillen <alejandro@alantechnologies.com>
+- Colton Donnelly <colton@alantechnologies.com>
+
+### Implementation
+
+- [ ] Implemented: [One or more PRs](https://github.com/alantech/alan/some-pr-link-here) YYYY-MM-DD
```

Comment:
This should probably be a milestone or two?

Reply:
I'm hesitant to break it up since as you can see from the diagram at the bottom of the RFC that there's a *lot* here and multiple ways we could slice it up. Even just the cluster status table may take multiple PRs, so I didn't want to state "must be one PR" in here.

Reply:
Got it, that makes sense